src/state/data-models/actions.js

```diff
@@ -0,0 +1,21 @@
+import { SET_DATA_MODELS_PAGED } from 'state/data-models/types';
+import { getDataModelsPaged } from 'api/dataModels';
+import { addErrors } from 'state/errors/actions';
+import { setPage } from 'state/pagination/actions';
+
+export const setDataModelsPaged = dataModelsPaged => ({
```

Comment:
Please remove "Paged" as it is not needed here. You are sending no information about paging with this action, this can bring confusion.